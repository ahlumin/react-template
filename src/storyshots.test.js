
/**
 * Take reference about the configuration API, see below:
 * https://github.com/infinitered/addon-storyshots
 */

import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';
initStoryshots();