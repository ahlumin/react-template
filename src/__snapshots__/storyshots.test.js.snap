// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Home a normal component 1`] = `
<div
  className="main"
>
  From App Yo.!
</div>
`;
